content/press/press-content.md

```diff
@@ -1,11 +1,34 @@
 <!--This is the main content file to edit for this page. It is embedded in src/pages/press.js-->
 
-**Press:** [Contact](info@environmentalenforcementwatch.org) a member of EDGI's EEW project team.
 
-## Images from events
+## Resources for press/media personnel
+**For inquiries and information:** [Contact](info@environmentalenforcementwatch.org) a member of EDGI's EEW project team.
+
+**Images from events**
 ![EEW public workshop participants](./eew-public-workshop.png)
 
-## Logos
+**Logos**
```

Comment:
If you choose to leave it bold instead of header, you need to add an extra carriage return between this and the next line, Markdown parses paragraph text that's spaced by one instead of two enters as single-line (so, as is, the image renders in line with the word "Logos"